circle.yml

```diff
@@ -39,3 +39,23 @@ deployment:
           -l "${CIRCLE_ARTIFACTS}"/acceptance_deploy 2>&1 >
           "${CIRCLE_ARTIFACTS}/acceptance_deploy.log"
       - build/push-aws.sh
+  osx:
+    branch: master
+    commands:
+      - |
+          if [ -n "${BUILD_OSX-}" ]; then
+            set -eux
+            # There must certainly be a better way to move ~/cockroach
```

Comment:
No, the builder doesn't have Docker, so we need to integrate https://github.com/jpetazzo/dind/blob/master/Dockerfile (which works), but then that has another magic wrapper script https://raw.githubusercontent.com/jpetazzo/dind/master/wrapdocker; I am just testing this locally and it looks like it would probably work (if the `--privileged` flag works on CI), but there are various warnings.

I still don't think we should go down that route - it seems prone to surprising and intransparent breakage. So, what speaks against just moving `~/cockroach` in its "correct" location for all the tests?